foxpuppet/windows/browser/navbar.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from foxpuppet.region import Region
+
+
+class NavBar(Region):
+
+    """Representation of the NavBar
+    :param selenium: WebDriver Object
```

Comment:
A `Region` object expects a `Window` object as the first argument. Note that this is different from the implementation in PyPOM, which is for web pages/regions.